README.rst

```diff
@@ -95,15 +95,15 @@ comm-3-bouncer      bouncer-prod-relengworker-firefoxci-comm-3
 pushapkscript
 -------------
 
-=================== ============================================
-Worker type         Deployment name
-=================== ============================================
-gecko-1-pushapk-dev pushapk-dev-relengworker-firefoxci-gecko-1
-gecko-3-pushapk     pushapk-prod-relengworker-firefoxci-gecko-3
-gecko-1-pushapk     pushapk-prod-relengworker-firefoxci-gecko-1
-mobile-3-pushapk    pushapk-prod-relengworker-firefoxci-mobile-3
-mobile-1-pushapk    pushapk-prod-relengworker-firefoxci-mobile-1
-=================== ============================================
+======================== ================================================
+Worker type              Deployment name
+======================== ================================================
+mobile-3-pushapk         pushapk-prod-relengworker-firefoxci-mobile-3
+mobile-1-pushapk         pushapk-prod-relengworker-firefoxci-mobile-1
```

Comment:
Isn't there a mobile-1-pushapk-dev pool?

Reply:
I thought we had one, but we don't. We can add it whenever we need it 🙂